website/content/en/docs/faqs/_index.md

```diff
@@ -4,6 +4,14 @@ linkTitle: FAQ
 weight: 10
 ---
 
+## What are the the differences between Kubebuilder and Operator-SDK?
+
+Kubebuilder and Operator-SDK are integrated projects which both allow you to quickly scaffold an operator project. Both tools make use [controller-runtime][controller-runtime] and the resulting projects will have the same basic layout. For more information see the doc about [what’s in a basic project][kb-doc-what-is-a-basic-project].  
+
+Operator-SDK offers additional features on top of the basic project scaffolding that Kubebuilder provides. It allows you to generate scaffolding to integrate your project with the [Operator Lifecycle Manager(OLM)][olm], a tool for managing the deployment, use, and upgrade of an operator on an actual in-use cluster. This scaffolding can also be used to distribute your operator on [OperatorHub][operatorhub.io], a community hub for publishing operators. Operator-SDK also offers the capability to test your deployed operator with [Scorecard][scorecard-doc], as well as the supporting other languages such as Ansible and Helm to create operators.
+
+For further context about the relationship between Kubebuilder and Operator-SDK, see [this blog post][operator-sdk-reaches-v1.0]. Note that in the past, the relationship was somewhat different, see [this comment][legacy-issue] about some of the legacy functionality from pre-1.0.0 versions of Operator-SDK.
```

Comment:
There's no need to reference legacy differences. It might confuse readers.
```suggestion
For further context about the relationship between Kubebuilder and Operator SDK, see [this blog post][operator-sdk-reaches-v1.0].
```